ui-tests/src/test/java/com/specmate/uitests/TestBase.java

```diff
@@ -11,120 +14,126 @@
-        capabilities.setCapability("deviceName", deviceName);
-        capabilities.setCapability("device-orientation", deviceOrientation);
-        capabilities.setCapability("platform", os);
-        capabilities.setCapability("tunnel-identifier", "github-action-tunnel"); 
-
-        String methodName = name.getMethodName();
-        capabilities.setCapability("name", methodName);
-
-        if (buildTag != null) {
-            capabilities.setCapability("build", buildTag);
-        }
-        this.driver = new RemoteWebDriver(new URL("https://" + username+ ":" + accesskey + seleniumURI +"/wd/hub"), capabilities);
-        
-        driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
-        driver.manage().timeouts().pageLoadTimeout(30, TimeUnit.SECONDS);
-
-        this.sessionId = (((RemoteWebDriver) driver).getSessionId()).toString();
-    }
-
-    @After
-    public void tearDown() throws Exception {
-        driver.quit();
-    }
-
-    public String getSessionId() {
-        return sessionId;
-    }
-
-    @BeforeClass
-    public static void setupClass() {
-        // Get the uri to send the commands to
-        seleniumURI = "@ondemand.saucelabs.com:443";
-        
-        // Set the buildTag to the Github Action number 
-        buildTag = "#" + System.getenv("GITHUB_RUN_NUMBER");
-        if (buildTag == null) {
-            buildTag = System.getenv("SAUCE_BUILD_NAME");
-        }
-    }
-    
+	public static String accesskey = System.getenv("SAUCE_ACCESS_KEY");
+	public static String seleniumURI;
+	public static String buildTag;
+	public static final String tunnelidentifier = System.getenv("TRAVIS_JOB_NUMBER");
+
+	public SauceOnDemandAuthentication authentication = new SauceOnDemandAuthentication();
+
+	/** Mark the Sauce Job as passed/failed when the test succeeds or fails */
+	@Rule
+	public SauceOnDemandTestWatcher resultReportingTestWatcher = new SauceOnDemandTestWatcher(this, authentication);
+
+	@Rule
+	public TestName name = new TestName() {
+		public String getMethodName() {
+			return String.format("%s", super.getMethodName());
+		}
+	};
+
+	protected String browser;
+	protected String os;
+	protected String version;
+	protected String deviceName;
+	protected String deviceOrientation;
+	protected String sessionId;
+	protected WebDriver driver;
+
+	/** Constructor for test instances */
+	public TestBase(String os, String version, String browser, String deviceName, String deviceOrientation) {
+		super();
+		this.os = os;
+		this.version = version;
+		this.browser = browser;
+		this.deviceName = deviceName;
+		this.deviceOrientation = deviceOrientation;
+	}
+
+	/** Browser configurations */
+	@Parameters
+	public static LinkedList<String[]> browsersStrings() {
+		LinkedList<String[]> browsers = new LinkedList<String[]>();
+
+		browsers.add(new String[] { "Windows 10", "88.0", "Chrome", null, null });
+		// browsers.add(new String[]{"Windows 10", "18.17763", "MicrosoftEdge", null,
```

Comment:
Please remove the commented code.

Reply:
I leave it there on purpose, because if we want to run on multiple browsers we can easily do it. But I can also remove it, since we are only using Chrome for years, because on the other the tests fails